consensus/wal.go

```diff
@@ -207,6 +207,10 @@ func (wal *BaseWAL) WriteSync(msg WALMessage) error {
 		return err
 	}
 
+	if err := wal.Write(msg); err != nil {
```

Comment:
Looks like patching the file led to this code being inserted twice (check line 206)

Reply:
Yep, I noticed later.

Reply:
Resolved